src/definition/api/IApi.ts

```diff
@@ -0,0 +1,82 @@
+import { IHttp, IModify, IPersistence, IRead } from '../accessors';
+import { IApiExample } from './IApiExample';
+import { IApiEndpointInfo } from './IEndpoint';
+import { IApiRequest } from './IRequest';
+import { IApiResponse } from './IResponse';
+
+export interface IApiEndpoint {
+    /**
+     * Path to complete the api URL. Example: https://{your-server-address}/api/apps/public/{your-app-id}/{path}
+     * or https://{your-server-address}/api/apps/private/{your-app-id}/{private-hash}/{path}
+     */
+    path: string;
```

Comment:
What would you think about allowing param style route definitions? For example: `/contacts/:contactId/numbers` would result in the passed in `contactId` being a param. Maybe this can be a future plan for improvements

Reply:
This already works 😄

Reply:
@rodrigok sweet! Maybe you can add some documentation so other developers are aware of it and can make usage of it 😃